components/panel/body.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+
+/**
+ * Internal dependencies
+ */
+import PanelBodyToggle from './body-toggle';
+
+class PanelBody extends Component {
+	constructor( props ) {
+		super( ...arguments );
+		this.state = {
+			opened: props.initialOpen === undefined ? true : props.initialOpen,
+		};
+		this.toggle = this.toggle.bind( this );
+	}
+
+	toggle( event ) {
+		event.preventDefault();
+		this.setState( {
+			opened: ! this.state.opened,
+		} );
+	}
+	render() {
+		const { toggle, children } = this.props;
+		return (
+			<div className="components-panel__body">
+				{ !! toggle &&
```

Comment:
At a glance it wasn't clear to me what `toggle` represented. Found later it's the label. Should we call it something more along those lines? `label`, `title`, `toggleLabel`, `heading`?